### ISSUE TITLE
Type II pneumocyte hyperplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6249]
**Preferred term label:**
Type II pneumocyte hyperplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Need to improve annotations for diseases associated with surfactant deficiency, e.g. https://omim.org/entry/610921

**Your nano-attribution (ORCID)**

**Comment by GrieseAG** [comment]
**Synonyms:** Increased level of type II pneumocytes
NEW Parent term: Abnormality of alveolar type II pneumocytes
**Diseases characterized by this term ?** OMIM: 263000 DIP

**Comment by pnrobinson** [comment]
Adding with definition
Enlargement of type II pneumocytes characterized by qualitative morphologic alterations, including cuboidal shapes, increased nucleocytoplasmic ratio, enlarged nuclei, prominent nucleoli, and various alterations in their nuclear chromatin. PMID:11079017

@GrieseAG  please reopen this issue if the definition needs more work.
